testbot.py

```diff
@@ -212,38 +211,41 @@ async def join(self, ctx, *, channel: discord.VoiceChannel):
 
     @commands.command()
     async def shibe(self, ctx):
-        await ctx.send(
-            "https://www." + str(get_video("shibes")).replace("\"", ""))
+        await ctx.send("https://www." + str(get_video("shibes")).replace('"', ""))
 
     @commands.command()
-    async def deepdream(self, ctx):
-        imagepath = './imagesdb/IMAGE' + str(ctx.message.id)
-        aiurl = "https://api.deepai.org/api/deepdream"
-        aifiles = {'image': aiofiles.open(imagepath, 'rb')}
-        aiheaders = {'api-key': '485f6ea6-1175-428f-9bff-43e04ee8fa09'}
-
+    async def dogedream(self, ctx):
+        """
+        Doge Dream command for helping Ziad
+        """
         try:
             url = ctx.message.attachments[0].url
         except Exception:
-            await ctx.channel.send(
-                "WHAT THE FUCK IS THIS??? USE A PROPER IMAGE YOU WANKER!!!!")
-            return
-        async with aiohttp.ClientSession() as session:
-            async with session.get(url) as resp:
-                if resp.status == 200:
-                    f = await aiofiles.open(
-                        imagepath, mode='wb')
-                    await f.write(await resp.read())
-                    await f.close()
-                    await ctx.channel.send(
-                        "OK FUCK I SAVED YOUR FUCKING IMAGE PLEASE STOP "
-                        "TORMENTING ME OH GOD OH FUCK")
-                else:
-                    raise Exception(
-                        'FUCKING IMAGE ISN"T THERE FUCKING FIX IT DIPSHIT FUCK')
-            async with session.post(aiurl, data=aifiles, headers=aiheaders) \
-                    as resp:
-                await ctx.channel.send(resp.)
+            ctx.send("Error! please use attachments")
+            # Note: you can pass url from message.content, I suggest using mimetypes to indentify if URL has image.
```

Comment:
Oh I see. Yeah I couldn't find a single source as to how you should get images and URLs from messages. And yeah I guess it's a good idea to check inputs to make sure it's an image. Almost like that's basic software dev or something hmm.